### PR TITLE
[MWAN, MT] MTU/MSS ref page and advice on settings

### PR DESCRIPTION
**Pull request by marciocloudflare**
### Summary

- Revamps get started part on MSS clamping
- Adds ref page with details on what MTU/MSS is, how CF handles it and advice on how to set up
- Closes PCX-11207

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

File: src/content/partials/magic-transit/mtu-mss/mss-clamping-gre.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+The MSS value depends on how your network is set up.
+
+- **Magic Transit ingress-only traffic (DSR):**
+
+  - **On your edge router transit ports**: Apply a TCP MSS clamp with a maximum of 1,436 bytes.
+  - **On any IPsec/GRE tunnels with third parties on your Magic Transit prefix**: Apply the MSS clamp on the internal tunnel interface (most likely on a separate firewall behind the GRE-terminating router) to reduce the current value by 24 bytes.
+
+- **For Magic Transit ingress + egress traffic**
```
Comment (Oxyjun) [inline src/content/partials/magic-transit/mtu-mss/mss-clamping-gre.mdx]:
```suggestion
- **For Magic Transit ingress + egress traffic:**
```

File: src/content/partials/magic-transit/mtu-mss/mtu-mss.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## MTU and MSS
+
+The [maximum transmission unit (or MTU)](https://www.cloudflare.com/learning/network-layer/what-is-mtu/) is a measurement representing the largest data packet that a network-connected device will accept. The MTU almost always applies to Layer 3 of the OSI model in networking, and includes the entire packet, including all headers (TCP, IP, etc.) and the data (payload) itself. For example, packets must be no bigger than 1,500 bytes to be routable through the Internet.
+
+The [maximum segment size (or MSS)](https://www.cloudflare.com/learning/network-layer/what-is-mss/) refers to the amount of data that can be sent in a single TCP datagram packet. This value is determined by subtracting the size of the IP and TCP headers from the MTU, which will instruct the router how large the payload can be. It applies to the Layer 4 of the OSI model in networking.
```
Comment (Oxyjun) [inline src/content/partials/magic-transit/mtu-mss/mtu-mss.mdx]:
```suggestion
The [maximum segment size (or MSS)](https://www.cloudflare.com/learning/network-layer/what-is-mss/) refers to the amount of data that can be sent in a single TCP datagram packet. This value is determined by subtracting the size of the IP and TCP headers from the MTU, which will instruct the router how large the payload can be. It applies to Layer 4 of the OSI model in networking.
```

File: src/content/docs/magic-transit/get-started.mdx [hunk 3]
```diff
@@ -54,9 +53,38 @@ You can also use the Resource Public Key Infrastructure (RPKI) as an additional
...
+
+:::caution[Important]
+Refer to your device documentation to check if it sets IPsec MSS clamping automatically. If that is not the case and you are using IPsec inside GRE, you have to set MSS clamp manually.
+:::
+
+Refer to [Maximum transmission unit and maximum segment size](/magic-transit/reference/mtu-mss/) for more details.
+
+#### Clear Do not fragment (DF)
+
+If you are unable to set the MSS on your physical interfaces to a value lower than 1500 bytes, you can choose to clear the `don't-fragment` bit in the IP header. When this option is enabled, Cloudflare fragments packets greater than 1500 bytes, and the packets are reassembled on your infrastructure after decapsulation. In most environments, enabling this option does not have significant impact on traffic throughput.
```
Comment (Oxyjun) [inline src/content/docs/magic-transit/get-started.mdx]: In some of the later files there are references to `do not fragment`. Sanity checking that this is specifically `don't-fragment` :D

File: src/content/partials/magic-transit/mtu-mss/mtu-mss.mdx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+![A diagram showing a UDP datagram and its various components.](~/assets/images/magic-transit/mtu-mss/udp-datagram.png)
+
+Suppose that the UDP datagram has source port `389` and is destined for a {props.productName} customer IP address. Let us also suppose that the {props.productName} customer has a firewall rule in place that drops UDP traffic with source port `389`, a common [Connectionless Lightweight Directory Access Protocol (CLDAP)](https://blog.cloudflare.com/reflections-on-reflections) reflection attack vector. The three packet fragments above will arrive at Cloudflare, but only the first fragment contains a UDP header with source port information — the second and third fragments, while they contain UDP data, do not have UDP header information. So the question is: which of these fragments does Cloudflare drop and which are delivered to the customer? If we only drop the first parts of fragmented packets, the remaining parts could still generate a large amount of traffic during a DoS attack.
+
+### How Cloudflare actually handles fragments
```
Comment (Oxyjun) [inline src/content/partials/magic-transit/mtu-mss/mtu-mss.mdx]:
```suggestion
### How Cloudflare handles fragments
```

File: src/content/partials/magic-wan/mtu-mss/mss-clamping-ipsec.mdx [hunk 5]
```diff
@@ -0,0 +1,7 @@
+---
+{}
+---
+
+For IPsec tunnels, the value you need to specify depends on how your network is set up. The MSS clamping value will be lower than for GRE tunnels, though, since the physical interface will see IPsec-encrypted packets, not TCP packets, and MSS clamping will not apply to those.
```
Comment (Oxyjun) [inline src/content/partials/magic-wan/mtu-mss/mss-clamping-ipsec.mdx]: For me, "though" here threw me off the cadence of the sentence. I read it as "although".

**Review by Oxyjun — APPROVED** [review]

File: src/content/docs/magic-transit/get-started.mdx (hunk 3 above)
Comment (marciocloudflare) [reply src/content/docs/magic-transit/get-started.mdx]: It's an old reference that escaped my changes. Thanks!

**Pull request merged**